src/Routes/SeasonsController.ts

```diff
@@ -16,7 +25,24 @@ export class SeasonsController extends BaseController {
                 BaseController._OnError(ex, res);
             }
         });
-
+/**
+ * @swagger
+ *
+ * /[seasonId]:
+ *   get:
+ *     description: Returns a list of Structs in a Season
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: id
+ *         description: Season ID
+ *         in:  url
+ *         required: true
+ *         type: string
+ *     responses:
```

Comment:
Also add the error case response when the given season id is not found, it would return a 404 code with the body having: {"errorCode":"NotFoundError","message":"Unable to find Season with id '123'"}

Reply:
done